packages/flutter/test/widgets/scrollable_semantics_test.dart

```diff
@@ -611,3 +656,13 @@ Future<void> fling(WidgetTester tester, Offset offset, int repetitions) async {
     await tester.pump(const Duration(seconds: 5));
   }
 }
+
+Rect nodeGlobalRect(SemanticsNode node) {
```

Comment:
Is there an existing function which already handles this? If not, should I move this somewhere more centralized?

Reply:
I don't think this is needed very often. Usually you only care about their relative position to the parent. Having that just here seems fine.